x11-misc/polybar/polybar-3.2.1.ebuild

```diff
@@ -20,7 +20,7 @@ LICENSE="MIT"
 SLOT="0"
 KEYWORDS="amd64 x86"
 
-IUSE="alsa curl i3wm ipc mpd network pulseaudio"
+IUSE="alsa bash-completion curl i3wm ipc mpd network pulseaudio zsh-completion"
```

Comment:
NAK, we don't use useflags to control tiny file installation anymore.
There may still be examples of this in the tree and global flags still present, but all new ebuilds should install those files unconditionally. someday someone will remove those useflags completely. But for now, let's skip them.

users can define `INSTALL_MASK` to avoid those small files.